src/App.js

```diff
@@ -1,7 +1,8 @@
 import * as React from "react";
 import { NavigationContainer } from "@react-navigation/native";
 import { createStackNavigator } from "@react-navigation/stack";
-import { HomeScreen, SignUpScreen } from "./pages";
+import { HomeScreen, SignUpScreen, ProfileScreen } from "./pages";
+// import { ProfileScreen } from "./pages/Profile/Profile";
```

Comment:
remove this line

Reply:
solve in d64f27a4351abcc9d759dbb9d55e3b8f238d6006